asio/src/doc/using.qbk

```diff
@@ -292,6 +292,48 @@ Asio.
         the map.
     ]
   ]
+  [
+    [`ASIO_HAS_MOVE`]
+    [
+      Move support is automatically enabled for g++ 4.5 and later, when the
+      `-std=c++0x` or `-std=gnu++0x` compiler options are used.
+
+      `ASIO_HAS_MOVE` explicitly enables move support for other/older
+      compilers.
+    ]
+  ]
+  [
+    [`ASIO_DISABLE_MOVE`]
+    [
+      Explicitly disables move support for all compilers.
+    ]
+  ]
+  [
+    [`ASIO_NO_DEPRECATED`]
+    [
+      Enables compile-time detection of the deprecated `asio_handler_invoke`
```

Comment:
What about `asio_handler_allocate` and `asio_handler_deallocate` (e.g. [asio/detail/wrapped_handler.hpp](https://github.com/chriskohlhoff/asio/blob/master/asio/include/asio/detail/wrapped_handler.hpp#L208))?

Reply:
@mabrarov You're right there were 2 bullet points in the changelog. I've merged them into one explanation for the macro.